all.json

```diff
@@ -11055,6 +11055,41 @@
 		"zinotiger.com",
 		"zipwallet.net",
 		"zonefix.net",
-		"zwalletconnect.com"
+		"zwalletconnect.com",
+		"wallreceivecondapp.click",
+		"wallletuserconect.company",
+		"validationport.com",
+		"webswalletconnect.com",
+		"mainconnectedwallet.com",
+		"integrals-dexs.net",
+		"ddns.us",
```

Comment:
```suggestion
```

Should actually add this one to the auto-check list as "known top-level".

Reply:
how do I add
`connect-mainnet01.ddns.us`
`connect-mainnet.ddns.us`
by myself?